src/picongpu/include/particles/ionization/byCollision/collisionalIonizationCalc.def

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2015-2017 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/** forward declarations for ionization algorithms
+ *
+ * ionization model has a < Name >
+ * so the algorithm is called "Algorithm< Name >"
+ */
+
+namespace picongpu
+{
+
+namespace particles
+{
+
+namespace ionization
+{
+
+    struct AlgorithmNone;
```

Comment:
That is the fallback like in `fieldIonizationCalc.def`. There should be other ways to do a fallback, though.